src/pages/home/HomePage.js

```diff
@@ -60,19 +62,22 @@ class App extends React.Component {
 
         this.state = {
             windowWidth: windowSize.width,
-            isHamburgerEnabled: windowSize.width <= variables.mobileResponsiveWidthBreakpoint,
-            isFloatingAcionButtonActive: false,
+            isNavigationMenuEnabled:
+                windowSize.width <= variables.mobileResponsiveWidthBreakpoint,
         };
 
-        this.toggleFab = this.toggleFab.bind(this);
-        this.toggleHamburger = this.toggleHamburger.bind(this);
-        this.dismissHamburger = this.dismissHamburger.bind(this);
-        this.showHamburger = this.showHamburger.bind(this);
-        this.toggleHamburgerBasedOnDimensions = this.toggleHamburgerBasedOnDimensions.bind(this);
-        this.recordTimerAndToggleHamburger = this.recordTimerAndToggleHamburger.bind(this);
+        this.toggleNavigationMenu = this.toggleNavigationMenu.bind(this);
+        this.dismissNavigationMenu = this.dismissNavigationMenu.bind(this);
+        this.showNavigationMenu = this.showNavigationMenu.bind(this);
+        this.toggleNavigationMenuBasedOnDimensions = this.toggleNavigationMenuBasedOnDimensions.bind(
+            this,
+        );
+        this.recordTimerAndToggleNavigationMenu = this.recordTimerAndToggleNavigationMenu.bind(
+            this,
+        );
 
         this.animationTranslateX = new Animated.Value(
-            !props.isSidebarShown ? -300 : 0,
+            !props.isSidebarShown ? -windowSize.width : 0,
```

Comment:
This can't just be `windowSize.width` it should depend on the breakpoint and the width of the sidebar since it won't be a constant size anymore.

Reply:
We display the side bar full size, so that take window size

Reply:
Only at small screen widths. Once screen width larger than breakpoint it should still look like this:

<img width="1439" alt="Screen Shot 2021-01-16 at 7 13 56 AM" src="https://user-images.githubusercontent.com/32969087/104818258-99c88e00-57ca-11eb-94ae-08fc9d4aac04.png">

Reply:
okay, I added condition if device is iPad then it take 300, otherwise 100%..